engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -122,6 +122,12 @@
     private final ShaderManager shaderManager;
     private final Camera playerCamera;
 
+    /*
+    * presumably, the eye height should be context.get(Config.class).getPlayer().getEyeHeight() above the ground plane.
+    * It's not, so for now, we use this factor to adjust for the disparity.
+     */
+    private final float GROUND_PLANE_HEIGHT_DISPARITY = -0.7f;
```

Comment:
Its odd,  this value is the negative value of PlayerConfig.DEFAULT_PLAYER_EYE_HEIGHT.   Are we doing too much math for these eye height values?